mypy/build.py

```diff
@@ -1977,6 +1986,19 @@ def _patch_indirect_dependencies(self,
             elif dep not in self.suppressed and dep in self.manager.missing_modules:
                 self.suppressed.append(dep)
 
+    def find_fine_grained_deps(self) -> None:
```

Comment:
Would you mind renaming this so the name more clearly implies that it has a side-effect?